server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -75,13 +96,13 @@ public void GET_to_request_age_25_users() throws IOException {
    * we get a reasonable error code back.
    */
   @Test
-  public void GET_to_request_users_with_illegal_age() {
+  public void respondAppropriatelyToIllegalAge() {
```

Comment:
Okay, full disclosure, this one really doesn't matter 🙃 but we may want to make all of the test methods declarative sentence fragments:

```suggestion
  public void respondsAppropriatelyToIllegalAge() {
```